py/torch_tensorrt/dynamo/conversion/_ConverterRegistry.py

```diff
@@ -404,7 +410,7 @@ def __getitem_without_validation__(
     def __getitem__(
         self, node: Node
     ) -> Tuple[
-        Any, CallingConvention
+        Any, CallingConvention, bool
```

Comment:
Lets make this like an info sort of thing `Any, CallingConvention, Dict[str, bool]` and put both dynamic shape and needs_output_allocator in there